package-lock.json

```diff
@@ -4195,7 +4195,6 @@
       "version": "4.4.0",
       "resolved": "https://registry.npmjs.org/@eslint-community/eslint-utils/-/eslint-utils-4.4.0.tgz",
       "integrity": "sha512-1/sA4dwrzBAyeUoQ6oxahHKmrZvsnLCg4RfxW3ZFGGmQkSNQPFNLV9CUEFQP1x9EYXHTo5p6xdhZM1Ne9p/AfA==",
```

Comment:
A bunch of packages, including eslint-related ones, are changing from being dev dependencies to non-dev. That seems wrong on the surface but I'm not immediately sure the fix.

Maybe the eslint peerDependency of the new package is being satisfied automatically for existing packages by installing it as a non-dev dependency? If that's the issue then one possible fix could be explicitly listing eslint as a devDependency in existing packages. But I'm not sure if that's aligned with best practices (we don't explicitly depend on eslint in many of our NI package.jsons). Or maybe we need to keep the NI styleguide listed as a devDependency?

Or maybe it's not a problem?

Could you explore docs / other repos / impacts of different package configurations? @rajsite might have opinions or ideas too.

Reply:
Don't think its a problem that the lockfile expresses it this way. The best choices today are:
- the eslint shared configuration package mark them as peerDependency
- users of the shared config in the monorepo mark it as devDependency (although I really want npm to make a devPeerDependency concept to support the monorepo use-case. All eslint configs should be "peer" for npm installl in the monorepo at dev but should not be a production peer).